src/actions/firestore.js

```diff
@@ -97,6 +97,9 @@ export function get(firebase, dispatch, queryOption) {
         payload: (snap) => ({
           data: dataByIdSnapshot(snap),
           ordered: orderedFromSnap(snap),
+          fromCache:
+            (snap.metadata && typeof snap.metadata.fromCache === 'boolean') ||
+            true,
```

Comment:
I think it still doesn't work as expected.

```
const metadata = { fromCache: false } 
const snap = { metadata }

(snap.metadata && typeof snap.metadata.fromCache === 'boolean') || true
->
(metadata && true) || true
->
true || true
->
true
```

But I assume you would expect it to resolve to `false`.

Reply:
Maybe you meant `typeof snap.metadata?.fromCache === 'boolean' ? snap.metadata.fromCache : true`?

Reply:
Right. I switched to Rafal's suggestion for an optional. 

<img width="444" alt="Screen Shot 2021-05-26 at 7 16 20 AM" src="https://user-images.githubusercontent.com/140163/119675907-46a6bd80-bdf2-11eb-93a6-7c688fb74f65.png">